src/msw/dcmemory.cpp

```diff
@@ -131,6 +131,16 @@ void wxMemoryDCImpl::DoSelect( const wxBitmap& bitmap )
     {
         m_oldBitmap = hBmp;
     }
+
+    // Remember content scale factor used by the bitmap: we don't use it
+    // ourselves, but this can be needed later for creating fonts of the
+    // correct size.
+    m_contentScaleFactor = bitmap.GetScaleFactor();
```

Comment:
When a user uses the scale factor, this doesn't have an effect anymore?

Reply:
I don't think there is a way to set `wxDCImpl::m_contentScaleFactor`, i.e. the user is not supposed to be able to do it, are they?

Reply:
I think I was confused with `wxGraphicsContext::SetContentScaleFactor()`.